src/components/Blog/PostLayout.tsx

```diff
@@ -3,16 +3,7 @@ import { FunctionComponent } from 'react'
 import { MDXRemote } from 'next-mdx-remote'
 import Link from 'next/link'
 
-import {
-    Alert,
-    Blockquote,
-    EmbeddedHubSpot,
-    Figure,
-    TableWrapper,
-    Video,
-    YouTube,
-    TrySourcegraph,
-} from '@components'
```

Comment:
I wish our prettier config kept these on separate lines. It helps with avoiding merge conflicts line by line.

Reply:
Maybe we should change our prettier rules 🤔  - Did something automatically break these lines?

Reply:
I'm guessing there's a config rule. I think if there are 2 or more then it should break to new lines for readability.

Reply:
@st0nebraker I'm guessing there's a config rule. I think if there are 2 or more then it should break to new lines for readability.

Reply:
Agreed, I like large imports from the same file being on multiple lines too